src/plugins/aprinterplugin.cpp

```diff
@@ -24,16 +24,15 @@ void AprinterPlugin::extractTemp(const QString &lastMessage)
     // ok T:185.4 /185.0 B:60.5 /60.0
     QStringList list = lastMessage.split(QChar::fromLatin1(' '));
     // T:185.4 - current temperature
-    PrinterStatus.extruderTemp = list[0].mid(2).toFloat();
+    printerStatus.extruderTemp = list[0].mid(2).toFloat();
     // /185.0 - target temperature
-    PrinterStatus.extruderTargetTemp = list[1].mid(1).toFloat();
+    printerStatus.extruderTargetTemp = list[1].mid(1).toFloat();
     // B:185.4 - current temperature
-    PrinterStatus.bedTemp = list[2].mid(2).toFloat();
+    printerStatus.bedTemp = list[2].mid(2).toFloat();
     // /60.0 - target temperature
-    PrinterStatus.bedTargetTemp = list[3].mid(1).toFloat();
+    printerStatus.bedTargetTemp = list[3].mid(1).toFloat();
 
-    qCDebug(APRINTER_PLUGIN) << "Extruder" << PrinterStatus.extruderTemp << "/" << PrinterStatus.extruderTargetTemp;
-    qCDebug(APRINTER_PLUGIN) << "Bed" << PrinterStatus.bedTemp << "/" << PrinterStatus.bedTargetTemp;
```

Comment:
Removed since they are redundant now do we need to see 
T: TEMP
Extruder TEMP: 
Bed : Temp
Bed T/T Extruder T/T 
every second for every heat up ?